packages/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -16,7 +16,26 @@
  apiVersion: v2
  appVersion: 0.46.0
  dependencies:
-@@ -59,19 +68,17 @@
+@@ -17,6 +26,8 @@
+ - condition: hardenedNodeExporter.enabled
+   name: hardenedNodeExporter
+   repository: file://./charts/hardenedNodeExporter
++  tags:
++  - hardenedNodeExporter
+ - condition: k3sServer.enabled
+   name: k3sServer
+   repository: file://./charts/k3sServer
+@@ -41,6 +52,9 @@
+ - condition: nodeExporter.enabled
+   name: prometheus-node-exporter
+   repository: file://./charts/prometheus-node-exporter
++  tags:
++  - nodeExporter
++  - hardenedNodeExporter
```

Comment:
I'd like to avoid using tags since this would be a breaking change (for someone who chose to disable nodeExporter before). See https://github.com/rancher/charts/pull/1221#discussion_r643402266.